src/Regression/MultipleRegression.cs

```diff
@@ -1,113 +1,137 @@
-﻿namespace AiDotNet.Regression;
+﻿using System;
+using System.Threading.Tasks;
+using MathNet.Numerics.LinearAlgebra;
```

Comment:
It is another minor issue but we moved all using statements to Helpers/UsingsHelper.cs so if you can please move the necessary using statements there.